### PR TITLE
(RM) Remove pipeline v1 unused functions and unit tests

### PR DESCRIPTION
**Pull request by lbesnard**

### CODE REVIEW

**Comment by lbesnard** [comment]
as part of https://github.com/aodn/PO-Backlog/issues/2468,
I removed all the different unused functions from the pipeline V1 which aren't used anymore.

The only ones left are using ```po_s3_del```

I did a thorough check, and checked the pipelines as well as the good working order of the po_s3_del command on the pipeline box.

**Comment by digorgonzola** [comment]
I still get loads of errors when I run these tests in an Ubuntu 20 docker environment with Python 3.8. Not sure if they are significant

File: lib/common/util.sh [hunk 1]
```diff
@@ -4,88 +4,6 @@
...
-
-    return $retval
-}
-export -f _bulk_index_operation
-
-# bulk index files using talend
-# $1 - directory to cd to before running harvester
-# $2 - base to index files with (prefix, such as IMOS/Argo)
-# $3 - file containing list of files to index
-index_files_bulk() {
```
Comment (mhidas) [inline lib/common/util.sh:41]: This function, along with `unindex_files_bulk` could occasionally still be useful.
You can get by just calling `$HARVESTER_TRIGGER` directly, but it won't automatically save the log.

**Review by mhidas — COMMENTED** [review]
Looks ok to me.
The bulk harvesting/unharvesting functions could be kept until there's an alternative that uses the pipeline infrastructure. I just did a quick check in the `pipeline-ubuntu20` box, and they do work under Ubuntu 20.

File: lib/common/util.sh (hunk 1 above)
Comment (lbesnard) [reply lib/common/util.sh]: I re-added them.

Comment (mhidas) [reply lib/common/util.sh]: :+1: Thanks.

**Review by mhidas — APPROVED** [review]

**Comment by lbesnard** [comment]
There was a deprecating error on the ubuntu 20 VM 
```
#############################                                                                                                                                                                                                                                           [101/4951]
#############################
Executing: 'lib/test/python/test_nc_checker_call.py'
#############################
/usr/local/lib/python3.8/dist-packages/tabulate.py:7: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.10 it will stop working
  from collections import namedtuple, Iterable
```

which can be fixed by upgrading the python system package:
```
sudo pip install tabulate --upgrade
```

**Comment by digorgonzola** [comment]
> udo pip install tabulate --upgrade

This caused me issues in my environment due to a conflict with `compliance-checker` wanting a specific version of `tabulate`.

I've created another branch with some changes to the requirements. Are you able to check? https://github.com/aodn/data-services/pull/1082

**Pull request merged**